src/coreComponents/linearAlgebra/unitTests/testKrylovSolvers.cpp

```diff
@@ -127,9 +127,9 @@ class KrylovSolverTest : public KrylovSolverTestBase< typename LAI::ParallelMatr
     this->precond.setup( this->matrix );
 
     // Set up vectors
-    this->sol_true.createWithGlobalSize( this->matrix.numGlobalCols(), MPI_COMM_GEOSX );
-    this->sol_comp.createWithGlobalSize( this->matrix.numGlobalCols(), MPI_COMM_GEOSX );
-    this->rhs_true.createWithGlobalSize( this->matrix.numGlobalRows(), MPI_COMM_GEOSX );
+    this->sol_true.create( this->matrix.numLocalCols(), MPI_COMM_GEOSX );
```

Comment:
This seems like a behavior change?

Reply:
Yes, I removed the ability to auto-partition a vector, it just didn't seem that useful to have - we pretty much always want out vectors partitioned in a very specific way (aligned with mesh/dofs/matrix rows).